Pearson-Advance/openedx-themes
# PADV 511 - Migrate Studio theme

## Ticket

https://agile-jira.pearson.com/browse/PADV-511

## Description

This PR aims to migrate the pearson-studio-theme to Olive. This only adds the dropdown functionality to filter by organization in studio. This PR is related to this other in the edx-platform https://github.com/Pearson-Advance/edx-platform/pull/98

## Changes Made

- [x] Add organizations dropdown

## How to test

- Change to the branch vue/PADV-511 in your edx-platform
- Activate the studio_home.enable_global_staff_optimization waffle switch in the studio admin page
- Enable the ORGANIZATION_APP feature in Studio
- You should be able to access the org_names_list variable inside the index.html and should contain a list with the organizations
- To test you can enable the pearson-studio-theme and verify that the dropdown contains the sites created

## Screenshots

![Screenshot from 2023-05-23 07-57-47](https://github.com/Pearson-Advance/openedx-themes/assets/62396424/d413cb58-fdad-418e-a204-6adf2a1a8992)

![Screenshot from 2023-05-23 07-57-53](https://github.com/Pearson-Advance/openedx-themes/assets/62396424/34e0f26c-9737-4c74-a6cd-3e90c6634979)

## Reviewers
